libsql-server/src/namespace/mod.rs

```diff
@@ -685,10 +733,18 @@ pub struct PrimaryNamespaceConfig {
 pub type DumpStream =
     Box<dyn Stream<Item = std::io::Result<Bytes>> + Send + Sync + 'static + Unpin>;
 
-fn make_bottomless_options(options: &Options, name: NamespaceName) -> Options {
+fn make_bottomless_options(
+    options: &Options,
+    namespace_db_id: Option<String>,
+    name: NamespaceName,
```

Comment:
Same here: with dedicated enum, we could also pass one param instead of pair of `namespace_db_id`, `name`.

Reply:
done.